tests/data_files/constant/bitstring-in-dn.pem

```diff
@@ -0,0 +1,51 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Please add a file `tests/data_files/constant/` explaining the purpose of the directory.

I'm not sure if `constant` is the right name. It describes a property of the files, but not their purpose. I would prefer a name that reflects the purpose. The purpose is that these are inputs to some parsing tests, so maybe `parse`? Although it depends which parsing tests we want to put there exactly (see my other comment).

Reply:
I am suffer from the name also. I think we need a readme also.

Reply:
Rename it to `parse_input` in #7652 . Is it better? @gilles-peskine-arm